site/content/cluster/binaries_cluster.md

```diff
@@ -0,0 +1,260 @@
+---
+linktitle: "Binaries"
+weight: 2
+---
+
+# Creating an M3 Cluster with Binaries
+
+This guide shows you the steps involved in creating an M3 cluster using M3 binaries, typically you would automate this with infrastructure as code tools such as Terraform or Kubernetes.
+
+{{% notice note %}}
+This guide assumes you have read the [quickstart](/docs/quickstart), and builds upon the concepts in that guide.
+{{% /notice %}}
+
+## M3 Architecture
+
+Here's a typical M3 deployment:
+
+<!-- TODO: Update image -->
+
+![Typical Deployment](/cluster_architecture.png)
+
+An M3 deployment typically has two main node types:
+
+-   **Coordinator node**: `m3coordinator` nodes coordinate reads and writes across all nodes in the cluster. It's a lightweight process, and does not store any data. This role typically runs alongside a Prometheus instance, or is part of a collector agent.
```

Comment:
Maybe mention something about statsd here? like `or is part of a collector agent such as statsD.`